plasma/root_chain/contracts/RootChain/RootChain.sol

```diff
@@ -81,7 +88,7 @@ contract RootChain {
         currentChildBlock = childBlockInterval;
         currentDepositBlock = 1;
         currentFeeExit = 1;
-        exitsQueue = new PriorityQueue();
+        exitsQueues[address(0)] = address(new PriorityQueue());
```

Comment:
I'd leave a comment here stating why do we treat ETH (`address(0)`) specially here - reason being that it is the only currency we support ATM, and in the future, creation of `PriorityQueue` should be taken care of on first exit (or deposit?)